src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -402,16 +407,16 @@ export async function zipCode(modulePath: string) {
     throwIfCancelled()
 
     const tempFilePath = path.join(os.tmpdir(), 'zipped-code.zip')
-    fs.writeFileSync(tempFilePath, zip.toBuffer())
+    await fsCommon.writeFile(tempFilePath, zip.toBuffer())
```

Comment:
`fsCommon.writeFile` writes in `utf-8` encoding... may need to look back at some of these replacements to ensure that is correct

Reply:
`fsCommon.writeFile` does string vs buffer distinction 👍